src/main/java/uk/gov/hmcts/reform/civil/utils/UnavailabilityDatesUtils.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.civil.utils;
+
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.Party;
+
+public class UnavailabilityDatesUtils {
+
+    private UnavailabilityDatesUtils() {
+                //NO-OP
+    }
+
+    public static void rollUpExpertUnavailabilityDates(CaseData.CaseDataBuilder<?, ?> builder, boolean defendantResponse) {
+        CaseData caseData = builder.build();
+        if (defendantResponse) {
```

Comment:
Minor: Is it not worthwhile performing this logis in the Defendant Response / Claimant Response handlers directly?
Or to split out this logic and have 2 util methods, one for each service. Saves us having to pass a boolean value to determine if its defendant or claimant